script/bulk_import/base.rb

```diff
@@ -484,7 +486,8 @@ def create_records(rows, name, columns)
         mapped = yield(row)
         next unless mapped
         processed = send(process_method_name, mapped)
-        imported_ids << mapped[:imported_id]
+        imported_ids << mapped[:imported_id] unless mapped[:imported_id].nil?
+        imported_ids |= mapped[:imported_ids] unless mapped[:imported_ids].nil?
```

Comment:
I mean multiple old categories into one new category. For clarity, here is a sample of my approach, which made my result very neat:
```yml
categories:
  - id: 1
    pre_existed: true
    is_uncategorized: true
  - id: 3
    pre_existed: true
    imported_ids: [4]
  - id: 4
    imported_ids: [27, 52]
    pre_existed: true
  - id: 5
    name: YAMAHA
    description: Discussion about YAMAHA keyboards.
    imported_ids: [76, 77, 78]
    children_categories:
      - id: 6
        name: External data
        description: YAMAHA keyboards' external data.
        imported_ids: [6, 7, 8, 46]
```